### ISSUE TITLE
Text area in editor missing ARIA label

### ISSUE THREAD

**Issue by daminihs** [wordplaydev/wordplay#688]
### What happened?

WCAG rule 2.1: Form elements (textbox/text area) must have a label
A label is missing from the text area in the editor.
Helpful article to resolve the issue: https://dequeuniversity.com/rules/axe/4.10/label?application=AxeChrome

To solve this problem, you need to fix at least (1) of the following:
-Element does not have an implicit (wrapped) <label>
-Element does not have an explicit <label>
-aria-label attribute does not exist or is empty
-aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-Element has no title attribute
-Element has an empty placeholder attribute
-Element's default semantics were not overridden with role="none" or role="presentation"

### Anything else?

![Image](https://github.com/user-attachments/assets/09f87c27-7c4f-4822-9e74-8ef380e500cd)

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows
